src/styles/_catalog.scss

```diff
@@ -1,10 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
 
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+
+  @media (max-width: 1024px) {
+    & {
```

Comment:
We can delete this row

Reply:
agree, fix @OlyaZaiets comment, this line is redundant